### PR TITLE
Render processing view for stalled uploads

### LINKED ISSUE #1094: Media processing errors vs timeouts
Right now, the UI doesn't distinguish between generic processing errors and timeouts.

image

Let's make that error message better (more specific) for timeouts!  Especially because we do periodically see these taking longer than whatever I set for the "stalled" time.  (90 seconds?)

### PR DESCRIPTION
**Pull request by radical-ube**
resolves #1094 

This PR solves a "hidden timeout" issue users were experiencing, by rendering a "still processing" message for "stalled" uploads. The solution i chose to solve for this was to opt for rendering the `_processing` partial instead of the `_error` partial for the `upload_stalled?` user conditional flow. I did this because i saw that the only difference between `upload_processing?` and `upload_stalled?` was the `retryable` condition. Therefore, a "stalled" upload should still be processing. A side benefit to using the `_processing` partial is that it continually polls the server, whereas the `_error` partial does not; if a "stalled" upload eventually completed, the `_error` view wouldn't refresh on its own to a `_complete` view, whereas the `_processing` view does.

Within the `_processing` partial, i added the `retry` button that is conditionally present in the `_error` partial, as well as a small (conditional) message underneath the field that notifies the user if the processing is taking longer than expected. This way, we keep the functionality for a user to choose to retry.

@cavis for functional review
@brandonhundt for language review

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -649,6 +649,7 @@ en:
         title: MP3 Not Provided
       processing:
         hint: Processing
+        still_processing: Processing is taking longer than expected. Thank you for waiting.
```
Comment (cavis) [inline config/locales/en.yml:652]: Nice, I do like how this isn't highlighted in red now.   Looks less angry:

image

I might prompt them that it's cool to click the retry button though... "Processing is taking longer than expected - you can continue waiting or retry"

**Review by cavis — CHANGES_REQUESTED** [review]
Makes good sense to me!

Searching through the code, it looks like Transcripts and Images both use basically identical retry logic to this.  Can we port this to those places as well?

File: config/locales/en.yml (hunk 1 above)
Comment (brandonhundt) [reply config/locales/en.yml:652]: FWIW: I prefer the term "patience" over "waiting", ie: "thank you for your patience"... Throw in a llittle "Ope" and we might even get the user to chuckle in another wise frustrating moment.

I suggest:
"Ope, file processing is taking longer than expected - you can continue waiting or retry"

**Review by cavis — APPROVED** [review]
:+3: nice.  The spinner being active really conveys that processing stalled, and it's not as angry as landing in error state.

**Pull request merged**